feat(getting-started): Switch from manual setup to sentry-wizard for Android and iOS in getting started (#7767)
getsentry/sentry-docs: src/platform-includes/getting-started-verify/android.mdx

<Note>

> If you're using [automatic insallation via the wizard](/platforms/android/#install), this step is not necessary.

Reviewer: ```suggestion
If you're using [automatic installation via the wizard](/platforms/android/#install), this step isn't necessary.
```
